fontforge/python.c

```diff
@@ -1162,7 +1150,7 @@ static PyObject *PyFF_FontTuple(PyObject *UNUSED(self), PyObject *UNUSED(args))
     for ( fv=FontViewFirst(), cnt=0; fv!=NULL; fv=fv->next, ++cnt );
     tuple = PyTuple_New(cnt);
     for ( fv=FontViewFirst(), cnt=0; fv!=NULL; fv=fv->next, ++cnt )
-	PyTuple_SET_ITEM(tuple,cnt,PyFV_From_FV_I(fv));
+	PyTuple_SET_ITEM(tuple,cnt,PyFF_FontForFV(fv));
```

Comment:
Unfortunately `PyFV_From_FV_I()` is a convenience routine that also increments the reference count (presumably what the "I" is for), so either we still need that convenience or calls to it need to be replaced with an explicit increment (the former is probably easier). 

However, it can just call `PyFF_FontForFV` now, or we can just call it `PyFF_FontForFV_I` and put it next to `PyFF_FontForFV`, since you've already done the other relevant editing anyway.

Reply:
Oh right, this is the same reason we needed to increment `Py_True`. Thanks, I'll do it.

Reply:
Um, @skef, why can't `PyFF_FontForFV` just do the increment, and we decrement where it's not needed? Incrementing is the more common case.

Reply:
(I pushed a version where `PyFF_FontForFV` always increments.)

Reply:
There are two separate issues here: the calling convention and the reference counts. 

I'm a little uncomfortable with this convention on a "that's not how this is typically done" basis, but I'm not sure I can make a principled argument. So I guess it's a preference. 

However, it is important to preserve the counts -- otherwise there will be (additional?) memory leaks. So if you change `PyFF_FontForFV()` to always increment, you have to change all the other callers to decrement. You can't (or shouldn't) just bump up the counts arbitrarily.

Reply:
There are three places we shouldn't increment, so I've decremented there as appropriate. :)